tests/base_test.py

```diff
@@ -140,6 +141,15 @@ def generate_token(payload):
         token = jwt.encode(payload, decoded_key, algorithm="RS256")
         return token
 
+    @staticmethod
```

Comment:
This is still in the `BaseTestcase`.

Reply:
Fixed. I have moved it to `ActivitiesTestCase`